src/main/java/mate/academy/spring/annotations/validators/RepeatPasswordValidator.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.annotations.validators;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.annotations.RepeatPassword;
+
+public class RepeatPasswordValidator implements ConstraintValidator<RepeatPassword, Object> {
+    @Override
+    public boolean isValid(Object dto, ConstraintValidatorContext context) {
```

Comment:
We can make it without reflection api: https://www.baeldung.com/spring-mvc-custom-validator